tests/service/context.rs

```diff
@@ -18,8 +18,8 @@ pub struct TestContext {
 impl TestContext {
     pub fn get_member(&mut self, network_id: String) -> Member {
         let mut member = Member::new(network_id, self.identity.clone());
-        if self.member_config.is_some() {
-            member.config = self.member_config.clone();
+        if let cfg @ Some(_) = self.member_config.clone() {
```

Comment:
I've never seen this before. Can you explain it?

Reply:
This just binds the whole `Option` object to the `cfg` variable, but only if it's the `Some` variant

Reply:
I guess this part can be off-putting to someone who sees it for the first time. If you don't like it we can revert it.

Reply:
well I guess what's the difference here between this and:

```rust
if let Some(cfg) = self.member_config.clone() 
```

Reply:
> well I guess what's the difference here between this and:
> 
> ```rust
> if let Some(cfg) = self.member_config.clone() 
> ```

here you bind the `cfg` to the thing inside the option, so then if you want to assign it to the member.config it ends up a bit more verbose: `member.config = Some(cfg);`

Reply:
ahh. I think I see now. Seems a little extra but sure, ok.